.drone.star

```diff
@@ -1123,50 +1128,51 @@ def e2eTests(ctx):
         "temp": {},
     }]
 
-    e2e_test_ocis = [{
-        "name": "e2e-tests",
-        "image": OC_CI_NODEJS % DEFAULT_NODEJS_VERSION,
-        "environment": {
-            "BASE_URL_OCIS": "ocis-server:9200",
-            "HEADLESS": "true",
-            "OCIS": "true",
-            "RETRY": "1",
-            "WEB_UI_CONFIG": "%s/%s" % (dirs["base"], dirs["ocisConfig"]),
-            "LOCAL_UPLOAD_DIR": "/uploads",
-            "API_TOKEN": "true",
-        },
-        "commands": [
-            "cd %s" % dirs["web"],
-            "sleep 10 && pnpm test:e2e:cucumber tests/e2e/cucumber/**/*[!.oc10].feature",
-        ],
-    }]
-
-    e2eTestsSteps = \
-        skipIfUnchanged(ctx, "e2e-tests") + \
-        restoreBuildArtifactCache(ctx, "ocis-binary-amd64", "ocis/bin/ocis") + \
-        restoreWebCache() + \
-        restoreWebPnpmCache() + \
-        tikaService() + \
-        ocisServer("ocis", 4, [], tika_enabled = True) + \
-        e2e_test_ocis + \
-        uploadTracingResult(ctx) + \
-        logTracingResults()
+    pipelines = []
 
     if ("skip-e2e" in ctx.build.title.lower()):
         return []
 
-    if (ctx.build.event != "tag"):
-        return [{
-            "kind": "pipeline",
-            "type": "docker",
-            "name": "e2e-tests",
-            "steps": e2eTestsSteps,
-            "depends_on": getPipelineNames([buildOcisBinaryForTesting(ctx)] + buildWebCache(ctx)),
-            "trigger": e2e_trigger,
-            "volumes": e2e_volumes,
-        }]
+    for index, path in enumerate(feature_paths):
+        e2eTestsSteps = \
+            skipIfUnchanged(ctx, "e2e-tests") + \
+            restoreBuildArtifactCache(ctx, "ocis-binary-amd64", "ocis/bin/ocis") + \
+            restoreWebCache() + \
+            restoreWebPnpmCache() + \
+            tikaService() + \
+            ocisServer("ocis", 4, [], tika_enabled = True) + \
```

Comment:
the tika service run for feature `fullTextSearch.ocis.feature`. And we know it runs on `e2e-tests-1` pipeline for index 1. Do we need to this service in the other pipe line as well?

Reply:
no! nice catch :+1: